### PR TITLE
Add missing SpigotConfig logCommands check

### LINKED ISSUE #10658: Commands are logged even when disabled
### Expected behavior

Commands should not be logged if disabled in spigot.yml (commands.log)

### Observed/Actual behavior

Commands are indeed logged

### Steps/models to reproduce

disable commands.log in spigot.yml
execute a command

### Plugin and Datapack List

[17:31:30 INFO]: Server Plugins (7):
[17:31:30 INFO]: Paper Plugins:
[17:31:30 INFO]:  - Commander, HologramAPI, Protect, Tweaks, Worlds
[17:31:30 INFO]: Bukkit Plugins:
[17:31:30 INFO]:  - FastAsyncWorldEdit, LuckPerms
> datapack list
[17:31:33 INFO]: There are 6 data pack(s) enabled: [trade_rebalance (feature)], [bundle (feature)], [vanilla (built-in)], [file/bukkit (world)], [file/tectonic.zip (world)], [paper (built-in)]
[17:31:33 INFO]: There are no more data packs available

### Paper version

git-Paper-37 (MC: 1.20.6) (Implementing API version 1.20.6-R0.1-SNAPSHOT) (Git: 156675c)

### Other

Fixed this issue once
https://github.com/PaperMC/Paper/pull/8624

### PR DESCRIPTION
**Pull request by NonSwag**
fixed the option commands.log in spigot.yml again
unsigned commands are logged even if command execution logging is disabled
fixes https://github.com/PaperMC/Paper/issues/10658

### CODE REVIEW

File: patches/server/1046-Add-missing-SpigotConfig-logCommands-check.patch [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+index 8e67853a7a93fa736c147e8b2df537746dc8e94f..5e9202bc7fc649764568b55d66ba0d684118c00c 100644
+--- a/src/main/java/net/minecraft/server/network/ServerGamePacketListenerImpl.java
++++ b/src/main/java/net/minecraft/server/network/ServerGamePacketListenerImpl.java
+@@ -2181,7 +2181,9 @@ public class ServerGamePacketListenerImpl extends ServerCommonPacketListenerImpl
+     private void performUnsignedChatCommand(String command) {
+         // CraftBukkit start
+         String command1 = "/" + command;
++        if (org.spigotmc.SpigotConfig.logCommands) { // Paper - Add missing SpigotConfig logCommands check
+         ServerGamePacketListenerImpl.LOGGER.info(this.player.getScoreboardName() + " issued server command: " + command1);
++        }
```
Comment (notTamion) [inline patches/server/1046-Add-missing-SpigotConfig-logCommands-check.patch]: can just remove the brackets since this is only a 1 line if statement

**Review by Machine-Maker — CHANGES_REQUESTED** [review]
This can be merged into the general logging changes patch that we have. Either author can do that or whoever merges this.

**Comment by NonSwag** [comment]
how do i do that ?

**Comment by notTamion** [comment]
there are multiple ways. you can just go inside Paper-Server and soft reset the commit you created, then stash your changes, rebase edit the commit like explained in the contributing.md, stash pop, and then just continue like the contributing.md says

**Comment by NonSwag** [comment]
Thanks, but I was more confused about the `general logging changes patch`
because i couldn't find anything like that

**Comment by notTamion** [comment]
i assume Machine is talking about https://github.com/PaperMC/Paper/blob/master/patches/server/0805-Improve-logging-and-errors.patch

**Review by Machine-Maker — APPROVED** [review]

**Review by lynxplay — APPROVED** [review]

**Pull request merged**